src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -167,13 +181,13 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 		&self,
 		"self",
 		"",
-		"Hint at this node's IP address from within the list passed in --ips",
+		"Optional IP address for redpanda to listen on; if empty, defaults to a private address",
 	)
 	c.Flags().IntVar(
 		&id,
 		"id",
 		-1,
-		"This node's ID. If unset, Redpanda will assign one automatically.",
+		"This node's ID. If unset, redpanda will assign one automatically",
```

Comment:
```suggestion
		"This node's ID. If unset, Redpanda assigns one automatically.",
```

Reply:
per DMs -- IIRC, our guidance currently is to use lowercase when referring to the process, and uppercase when referring to the company. We use redpanda pretty widely at the moment, so we'll keep the lowercase for now for consistency and for current guidance, but we can revisit wholesale in the future.